src/tokenizer.ts

```diff
@@ -174,14 +177,20 @@ export class Tokenizer {
         const lexeme = this.source.slice(this.start, this.current);
         this.tokens.push(new Token(type, lexeme, line, col, this.current - lexeme.length))
     }
-
+    // Ensures that lexeme value is parsed correctly by removing starting and ending quotes.
```

Comment:
Comments should be as close to the source as they are describing. In this case it should be placed on a separate line above the this.source.slice